packages/bigcommerce/src/api/operations/login.ts

```diff
@@ -63,19 +63,13 @@ export default function loginOperation({
         cookie = cookie.replace(/; SameSite=none/gi, '; SameSite=lax')
       }
 
-      const prevCookie = response.headers.get('Set-Cookie')
-      const newCookie = concatHeader(prevCookie, cookie)
```

Comment:
The import for `concatHeader` was not removed, and that util might not be needed anymore if it's no longer used here, also, why is this part changing?

Reply:
Since the res is not passed after dynamic handler PR, I removed that from here too.